bin/hdfcoinc/pycbc_coinc_mergetrigs

```diff
@@ -71,7 +71,10 @@ frpc = numpy.array([], dtype=numpy.float64)
 stf = numpy.array([], dtype=numpy.float64)
 gating = {}
 for filename in args.trigger_files:
-    data = h5py.File(filename, 'r')
+    try:
+        data = h5py.File(filename, 'r')
+    except IOError:
+        print 'Cannot open', filename
```

Comment:
We should try to avoid print statements in our code. Please use the logging module. If you use logging.warning it will appear even if verbose mode is not on.